android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -358,6 +360,28 @@ public void getSystemAvailableFeatures(Promise p) {
     p.resolve(promiseArray);
   }
 
+  @ReactMethod
+  public void hasLocationServicesEnabled(Promise p) {
+      boolean locationEnabled = false;
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT){
```

Comment:
Can you add the case for Build.VERSION_CODES.P and https://developer.android.com/reference/android/location/LocationManager.html#isLocationEnabled() also?

These APIs move so fast (to my way of thinking), but P does already have about 33% of the market, worth moving into their APIs as long as we check versions

Reply:
Hmm - now that I think about it, I wonder of having a return shape that wasn't a simple boolean might be interesting? For instance, a lot of apps are interested in what level of GPS is allowed - GPS-only, A-GPS, wifi only etc. What if there was a return shape that was an map, with name and enabled pairs? Then a wrapper that give you a simple boolean if you wanted, by scanning the main return map for any entry that had a value of true? Then the API on Android would be https://developer.android.com/reference/android/location/LocationManager.html#getProviders(boolean) and it is since API1